package.json

```diff
@@ -14,6 +14,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "crypto": "0.0.3",
```

Comment:
Not sure about using `crypto` here, since it looks to me that all other hashing algorithms will be included in the resulting bundle. Quick search brought me to [trentmillar/md5-o-matic](https://github.com/trentmillar/md5-o-matic) and [sindresorhus/md5-hex](https://github.com/sindresorhus/md5-hex). The second uses the first one in browser and resorts to native `crypto` package in Node. What do you think?